packages/containers/src/Notification/Notification.sagas.js

```diff
@@ -22,6 +22,9 @@ export function* onPushNotification(action) {
 }
 
 function* onCMFError(action) {
+	if (process.env.RELEASE_WITH_ERROR && process.env.NODE_ENV === 'production') {
```

Comment:
better named by RELEASE_WITHOUT_ERROR as it does a return ?

Reply:
or NOTIFICATION_JS_ERROR because I think that this variable disable the errors. It is only the notification.